code/platforms/linux/desktops.py

```diff
@@ -0,0 +1,25 @@
+from talon import Context, Module, actions, ui
+ctx = Context()
+ctx.matches = r"""
+os: linux
+"""
+
+@ctx.action_class("user")
+class Actions:
+    def desktop(number: int):
+        ui.switch_workspace(number)
+
+    def desktop_next():
+        ui.switch_workspace(ui.active_workspace() + 1)
```

Comment:
What happens if you're on the highest one here? Does it wrap around?

Reply:
Unfortunately I don't have a linux desktop right now, so all the linux work was best guess :-(. It probably crashes honestly, or does nothing. I figured if I can't test it, might as well write the smallest amount possible!

(I should probably get a VM or start dual booting so I can contribute better here!)

Reply:
On ubuntu at least, trying to switch_workspace() to a workspace that doesn't exist raises an error, which is basically correct behavior. so this is fine.